R/convert_docx.R

```diff
@@ -0,0 +1,74 @@
+#' @title Convert a docx-file (a protocol) to an (R)markdown file
+#'
+#' @description This function is derived from the \link[redoc]{dedoc} and uses pandoc to convert between docx and markdown. Several options are preset to end-up with a markdown document that is in syntax as close as possible to Rmarkdown files in RStudio. During conversion, graphics (e.g. png, jpg) will be extracted from the docx archive and placed in a folder `./media` and named `image1`, `image2`, etcetera.
```

Comment:
```suggestion
#' @description This function is derived from the [redoc::dedoc()] function and uses pandoc to convert between docx and markdown. Several options are preset to end-up with a markdown document that is in syntax as close as possible to Rmarkdown files in RStudio. During conversion, graphics (e.g. png, jpg) will be extracted from the docx archive and placed in a folder `./media` and named `image1`, `image2`, etcetera.
```

Reply:
Does this create a link in the helpfile and pkgdown website?

Reply:
I guess so...

Reply:
In the helpfile it will link to local install, on site it should go to pkgdown site, see [here](https://pkgdown.r-lib.org/articles/linking.html#across-packages).